vendor/composer/ClassLoader.php

```diff
@@ -313,29 +334,34 @@ public function loadClass($class)
      */
     public function findFile($class)
     {
-        // work around for PHP 5.3.0 - 5.3.2 https://bugs.php.net/50731
-        if ('\\' == $class[0]) {
-            $class = substr($class, 1);
-        }
```

Comment:
Symphony technically still is compatible with PHP 5.3. I think I'll wait for a major version before dropping that line...